api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -32,16 +31,8 @@
 import org.pmiops.workbench.leonardo.model.LeonardoGetRuntimeResponse;
 import org.pmiops.workbench.leonardo.model.LeonardoListRuntimeResponse;
 import org.pmiops.workbench.leonardo.model.LeonardoRuntimeStatus;
-import org.pmiops.workbench.model.Authority;
-import org.pmiops.workbench.model.EmptyResponse;
-import org.pmiops.workbench.model.ListRuntimeDeleteRequest;
-import org.pmiops.workbench.model.ListRuntimeResponse;
+import org.pmiops.workbench.model.*;
```

Comment:
Same comments as in your few previous PRs, you can follow https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit to setup Intellij , that will your life way easier....

Reply:
ah.. actually I already set it up as the doc said. I just don't know why it doesn't work...

Reply:
It might be the no2 step here:
https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit#heading=h.41gomlitnj0
`Enable the google-java-format plugin for any relevant projects
`